.bcr/metadata.template.json

```diff
@@ -0,0 +1,22 @@
+{
+  "homepage": "https://github.com/protocolbuffers/protobuf",
+  "maintainers": [
+    {
+      "github": "comius",
+      "name": "Ivo List"
+    },
+    {
+      "email": "alex@aspect.dev",
+      "github": "alexeagle",
+      "name": "Alex Eagle"
+    },
+    {
+      "email": "sahin@aspect.dev",
+      "github": "thesayyn",
+      "name": "Şahin Yort"
+    }
```

Comment:
Given that we're moving this to our repo, shouldn't we be the maintainers?

Reply:
sure, I can't see a list of who is on the team, please propose an edit?

Reply:
{
      "email": "sandyzhang@google.com",
      "github": "zhangskz",
      "name": "Sandy Zhang"
    },
    {
      "email": "mkruskal@google.com",
      "github": "mkruskal-google",
      "name": "Mike Kruskal"
    },
    {
      "email": "gberg@google.com",
      "github": "googleberg",
      "name": "Jerry Berg"
    }

would be a decent placeholder until we setup a group email for this

Reply:
thanks - FYI github has a feature where you can just "pencil" on my file and make the edit yourself, or, you can make changes inside a code suggestion like critique

Reply:
yea I tried that, but the only option seemed to be to commit the changes directly to your branch.  That seems much more heavy-handed than a "suggestion"

Reply:
There's a subtle button right above the textarea where you type a comment, it looks like a page with a plus/minus on it, that's where you can make suggested edits. But yeah this is fine too :)